web_accounts_list.json

```diff
@@ -132,6 +154,17 @@
         "category" : "finance",
         "valid" : true
       },
+      {
+        "name" : "carrd.co",
+        "check_uri" : "https://{account}.carrd.co",
+        "account_existence_code" : "200",
+        "account_existence_string" : "( Made with Carrd )",
+        "account_missing_code" : "404",
+        "account_missing_string" : "Sorry, the requested page could not be found.",
+        "known_accounts" : ["miltia"],

```

Comment:
Can you please add another `known_accounts` entry here? Always like having at least 2.